_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -328,9 +327,31 @@ public static function wrap_ui( $callback, $args = array() ) {
 			echo $callback_ui;
 			?>
 			<!-- END OF CALLBACK -->
-			<div class="jp-footer">
+			<?php self::render_footer(); ?>
+		</div>
+		<?php
+		return;
+	}
+
+	/**
+	 * Render the footer of the jetpack dashboard. For admin pages.
+	 */
+	function render_footer() {
+		$admin_url = admin_url( 'admin.php?page=jetpack' );
+
+		$is_connected = Jetpack::is_active() || Jetpack::is_development_mode();
+		$about_url = ( $is_connected )
+			? admin_url( 'admin.php?page=jetpack_about' )
+			: 'https://jetpack.com';
+
+		$privacy_url = ( $is_connected )
+			? $admin_url . '#/privacy'
+			: 'https://automattic.com/privacy/'
+
+		?>
+		<div class="jp-footer">
 				<div class="jp-footer__a8c-attr-container">
-					<a href="<?php echo esc_url( $jetpack_about_url ); ?>">
```

Comment:
Since we're getting rid of `$jetpack_about_url`, maybe we can remove where the variable is defined as well?